test/config.json

```diff
@@ -510,6 +499,28 @@
 			"Shard": 17,
 			"RetryMax": 0,
 			"Tags": []
+		},
+		"worker": {
+			"File": "worker.py",
+			"Args": ["--topo-server-flavor=etcd2"],
+			"Command": [],
+			"Manual": false,
+			"Shard": 23,
+			"RetryMax": 0,
+			"Tags": [
+				"worker_test"
+			]
+		},
```

Comment:
@saurabh408 We don't need to run python one can you please remove that? And also update the GO one to run in shard 23

Reply:
I ran it for test only.I reverted it.